package.json

```diff
@@ -93,11 +93,12 @@
     "marked-gfm-heading-id": "^4.1.1",
     "multer": "^1.4.5-lts.1",
     "nanoid": "^3.3.7",
-    "npm": "^11.0.0",
+    "npm": "^11.2.0",
```

Comment:
Didn't realise we had npm here... should this even be a dep of the project? NPM should be globally installed anywhere we're using it I think?

Reply:
No idea... it just updated itself though when I did an NPM install

Reply:
Let's remove it, shouldn't be there I think - possibly an accidental `npm i npm` instead of `npm i -g npm`